### ISSUE TITLE
Premature Failure - Error: "too many open files"

### ISSUE THREAD

**Issue by sdtsui** [wormi4ok/evernote2md#8]
On first run I get the following error - the export fails.

```
Notes: 83 / 2514 [=====>__________________________________________________________________________________________________________________________________________________________________________] 3.30%
1s2020/07/02 00:42:32 open [redacted].md: too many open files
Notes: 246 / 2514 [=================>_____________________________________________________________________________________________________________________________________________________________] 9.79% 7s
```

This is fixed by increasing your ulimit: 
```
sdtsuis-MacBook-Air:EverNoteExport sdtsui$ ulimit -n
256
sdtsuis-MacBook-Air:EverNoteExport sdtsui$ ulimit -n 3000
sdtsuis-MacBook-Air:EverNoteExport sdtsui$ ulimit -n
3000
```

Not sure if this is unexpected behavior, or if you'd like to include that somewhere in README, an FAQ, etc.  Thought I'd file here as others might not find this fix.

Great project! Very pleased it works - I'm especially grateful for the image support.  I can basically move off EverNote for good with very little fuss.  Thank you!!

**Comment by wormi4ok** [comment]
Hey @sdtsui!

Thanks for the kind words and for reporting the problem that you faced!

I think I can fix this issue. It went unnoticed because my `.enex` export for testing is not big enough to hit the limit. Expect to see a patched version this weekend. 

You would help a lot if you can test the patched version and leave a comment.

**Comment by wormi4ok** [comment]
Hi @sdtsui!

I released a new version - [0.7.0](https://github.com/wormi4ok/evernote2md/releases/tag/v0.7.0) that should not have an issue you mentioned. Could you please try to revert your `ulimit` value to 256 and convert the same `*.enex` file?

Let me know if it works without errors this time.

**Comment by sdtsui** [comment]
@wormi4ok - it was a one-time export for me and I've since deleted everything from that computer (both the *.enex and EverNote itself), so I don't think I'll be able to help test this release.

A friend of mine also did an export, and they were at around the same size (>1k notes, lots of images, >3yrs of usage) and it worked fine, so I think you should be good until the next person has an issue? :)

I'm sorry I can't be more helpful.  Thought I'd at least reply though, even if I can't test the release.  Again, awesome work!
